### PR TITLE
Update ug and dg

### LINKED ISSUE #113: Update DG formatting
* Update DG formatting
* Update MSS for headers

### LINKED ISSUE #114: Update User Profile in the UG
* Update the dash.
* Other description changes.

### PR DESCRIPTION
**Pull request by orzymandias**
* Update user profile command to include `[TITLE]` fields
* Update phrasing of user profile descriptions
* Update developer guide formatting

closes #113 
closes #114

### CODE REVIEW

**Review by orzymandias — COMMENTED** [review]
resolved!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -197,89 +197,98 @@ Guarantees: Editing of internship application is successful
...
+1.  User requests to delete a user profile item from the user profile.
 2.  InternHunter removes the user profile item from the user profile. </br>
     Use case ends.
 
 #### Extensions
 
-1a. InternHunter detects an invalid input. <br />
-  1a1. InternHunter displays an error message and informs the user of the valid input format. <br />
-  Use case resumes from step 1.
+  1a. InternHunter detects an invalid input. <br />
```
Comment (seanjyjy) [inline docs/DeveloperGuide.md]: hmm i am not sure which "style" we want to stick to but shawn has another way of writing this `InternHunter detects an error in the input format` . We might want to stick to one way in order for our DG to look cohesive.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -197,89 +197,98 @@ Guarantees: Editing of internship application is successful
...
+  1a1. InternHunter displays an error message and informs the user of the valid input format. <br/>
+  Use case resumes from step 1.
 
 ### Use case: UC13 - Switch tabs
 
 #### MSS
 
 1.  User requests to switch the tab of the screen.
-2.  InternHunter switches to the requested tab. <br />
+2.  InternHunter switches to the requested tab. </br>
```
Comment (seanjyjy) [inline docs/DeveloperGuide.md]: should use <br />

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -197,89 +197,98 @@ Guarantees: Editing of internship application is successful
...
 
 #### MSS
 
-Precondition: User already has an existing user profile items </br>
+Precondition: User is on the Profile page and already has an existing list of user profile items <br/>
 Guarantees: Deletion of user profile item is successful
 
-1.  User requests to delete a user profile item from the user profile and provides the index.
+1.  User requests to delete a user profile item from the user profile.
 2.  InternHunter removes the user profile item from the user profile. </br>
```
Comment (seanjyjy) [inline docs/DeveloperGuide.md]: should use <br/ >

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -197,89 +197,98 @@ Guarantees: Editing of internship application is successful
...
 #### MSS
 
-Precondition: User already has an existing list of user profile items </br>
+Precondition: User is on the Profile page and already has an existing list of user profile items <br/>
 Guarantees: Editing of user profile item is successful
 
-1.  User requests to edit the details of a user profile item and inputs the index and details.
-2.  InternHunter edits the user profile item from the user profile. </br>
+1.  User requests to edit the details of a user profile item and provides details.
+2.  InternHunter edits the user profile item in the user profile. </br>
```
Comment (seanjyjy) [inline docs/DeveloperGuide.md]: should use <br/ >

File: docs/UserGuide.md [hunk 5]
```diff
@@ -202,48 +202,61 @@ Example:
 
 ### User Profile
 
-#### Adding to user profile: `add -me`
+Note: You must be viewing the **Profile** page in order to execute these commands.
```
Comment (seanjyjy) [inline docs/UserGuide.md]: Would it be weird if it is "viewing" the profile page? To be cohesive to our DG, `You must be on the **profile** page ...` sounds more cohesive to our precondition

**Review by seanjyjy — COMMENTED** [review]
Overall looks great, some nits to fix

**Comment by seanjyjy** [comment]
LGTM good job.

**Pull request merged**